loklak.php

```diff
@@ -149,4 +149,41 @@ public function settings() {
 			return json_encode($request, true);
 		}
 	}
+
+	public function suggest($query, $count=null, $order=null, $orderby=null, $since=null, $until=null) {
+		$this->requestURL = 'http://localhost:9000/api/suggest.json';
+		$this->query = $query;
+		$this->count = $count;
+		$this->order = $order;
+		$this->orderby = $orderby;
+		$this->since = $since;
+		$this->until = $until;
+		if($query) {
+			$params = array('q'=>$this->query);
+		}
+		if ($count) {
+			$params['q'] = $params['q'] . ' count:'.$this->count;
+		}
+		if ($order) {
+			$params['q'] = $params['q'] . ' order:'.$this->order;
+		}
+		if ($orderby) {
+			$params['q'] = $params['q'] . ' orderby:'.$this->orderby;
+		}
+		if ($since) {
+			$params['q'] = $params['q'] . ' since:'.$this->since;
+		}
+		if ($until) {
+			$params['q'] = $params['q'] . ' until:'.$this->until;
+		}
```

Comment:
All the `$params` are individual parameters and are constructed in the URL with `&` unlike in `search` where they are stringified with `%20`, So each of the if conditions should push to the params array with a new param

so
```php
params['order']=$this->order;
params['count']=$this->count;
```
etc..,

With this code you can check the result for `$l->suggest('apple', '5')` and check if its working ?